src/javascript/app/pages/cashier/cashier.js

```diff
@@ -149,6 +162,7 @@ const Cashier = (() => {
                         State.getResponse('statement'),
                         State.getResponse('mt5_login_list')
                     );
+                    showP2PNote();
```

Comment:
can we rename this to setP2PVisibility or similar? since it's not necessarily showing it in all cases